zenoh/tests/routing.rs

```diff
@@ -0,0 +1,591 @@
+//
```

Comment:
`cargo test` as well as `cargo nextest` (which is used in the CI) run in parallel all the tests included in the same file. If I'm not mistaken, multicast scouting is enabled in the tests. Therefore all the tests (`two_node_combination`, `gossip`, `static_failover_brokering`, and `three_node_combination`) will discover each other at runtime and they will establish a connection. I believe would be better to disable multicast scouting in the tests.

Reply:
@Mallets, I believe the multicast scouting of all tests is disabled here (https://github.com/eclipse-zenoh/zenoh/pull/494/files#diff-afb052b60cf8a8762f980ef370030e39bab9b9311b0e9b86231a341a97d69d05R238)